src/css/base.scss

```diff
@@ -1,6 +1,128 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
+@import 'user';
 
-body {
-  background-color: $primary-background;
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Montserrat', sans-serif;
 }
+
+.main-section {
+  width: 100%;
+  height: 100vh;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+  url('../images/cherry-blossom-background-4.jpg')no-repeat center center;
+  background-size: cover;
+}
+
+.header-navbar {
+  display: flex;
+  width: 100%;
+  height: 10%;
+  background-color: rgb(50, 52, 53);
+  text-align: center;
+  align-items: center;
+}
+
+.header-navbar-button {
+  height: 100%;
+  width: 5%;
+  background-color: rgb(50, 52, 53);
+  border: none;
+  margin: 0 0 0 1%;
+}
+
+.lines {
+  width: 75%;
+  height: 4px;
+  background-color: rgb(211, 211, 211);
+  margin: 10px 0;
+  transition: 0.4s;
+}
+
+.header-heading {
+  font-size: 2.5em;
+  margin-left: 34%;
+  margin-top: auto;
+  color: rgb(211, 211, 211);
+  font-family: 'Gayathri', sans-serif;
+  font-weight: 100;
+}
+
+.display-section {
+  display: flex;
+  height: 90%;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.user-login-section {
+  display: flex;
+  height: 60%; 
+  width: 50%;
+  border-radius: 7px;
+  background: linear-gradient(rgb(176, 176, 176), rgba(255, 255, 255, 0.297));
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.user-login-header {
+  font-size: 1.5em;
+  color: #522D63;
+}
+
+.user-input-section {
+  display: flex;
+  width: 55%;
+  height: 50%;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-direction: column;
+}
+
+.user-input-bar {
+  width: 100%;
+  height: 20%;
+  border-radius: 7px;
+  padding-left: 10px;
+  outline: none;
+  border: none;
+}
+
+.user-password-bar {
+  @extend .user-input-bar;
```

Comment:
I don't know if you plan to add additional properties to .user-password-bar, if not you could just add that class onto the .user-input-bar selector and comma separate, the extend might be extra.

Reply:
I completely understand and will probably be refactoring this as the last step